addons/sale/models/product_product.py

```diff
@@ -88,6 +88,7 @@ class ProductAttributeCustomValue(models.Model):
     _name = "product.attribute.custom.value"
     _rec_name = 'custom_value'
     _description = 'Product Attribute Custom Value'
+    _order = 'attribute_value_id'
```

Comment:
This is OK separately but becomes not correct if you add a sequence on the attribute lines.

But this whole custom class it not very good anyway, it should link to a ptav and not to a pav I think.

Reply:
I agree on the ugliness of this class :-/.

The problem here is that because no order is defined:
- You choose two custom attributes on a product and fill them
- You reopen the wizard to change the first one
- The configurator deletes the first one and creates a new with the new values, which changes the order of the two attributes and therefore how they are displayed in the SOLine description.

It was a quick fix due to a request i got from the support. As custom attributes aren't really used in the matrix, a cleaner fix/refactoring of the class should maybe go into another task and PR.